test.html

```diff
@@ -164,6 +281,65 @@ <h2>Test drive</h2>
 		});
 }
 
+/**
+ * Loads a plugin including it css file
+ *
+ * @param  {string} plugin the plugin to load
+ * @type {Promise} the promise which resolves as soon as plugin is loaded
+ */
+function loadPlugin (plugin)
+{
+	if (!components.plugins[plugin]){
+		return;
+	}
+
+	var pluginPath = components.plugins.meta.path;
+
+	return new Promise(function (resolve) {
+		pluginPath = pluginPath.replace(/{id}/g, plugin);
+
+		$u.script(pluginPath + '.js', function() {
+			this.setAttribute('data-type', 'plugin');
+
+			var style = document.createElement('link');
+			style.setAttribute('rel', 'stylesheet');
+			style.setAttribute('href', pluginPath + '.css')
+			style.setAttribute('data-type', 'plugin');
+			document.body.appendChild(style);
```

Comment:
Not all plugins have associated CSS, so we should check if it does first.

Reply:
Indeed, but should I just add here `onerror` and silent the message in case css file is not found? I mean if it's not found - nothing critical will happen :)